runtime/doc/api.txt

```diff
@@ -1572,6 +1479,99 @@ nvim_unsubscribe({event})                                 *nvim_unsubscribe()*
     Parameters: ~
       • {event}  Event type string
 
+nvim__complete_set({index}, {opts})                     *nvim__complete_set()*
+    EXPERIMENTAL: this api may change in the future.
```

Comment:
```suggestion
    EXPERIMENTAL: this API may change in the future.
```
This actually applies to all these functions. Can we move them to their own section, with this caveat at the head?

Reply:
that can be a followup. this is just an incremental improvement compared to the current situation where these are prominently at the top of the section